deploy/terraform/modules/container_registry/main.tf

```diff
@@ -0,0 +1,40 @@
+resource "aws_ecr_repository" "ecr" {
```

Comment:
@nonparibus it's `$0.09 per image scan`

Reply:
AFAIK the other static analytics tools I pinged your way effectively duplicate this functionality.

But how often are we creating these images and then scanning them? Is it on PRs merging into `main` which modify the Terraform scripts in such a way as to result in new versions of these images only? Or is it every push on one of those PRs? Or are other PRs that don't touch Terraform getting caught up in this, as well?

If the cost is small then it may just be fine to include, sensible to leave on as an additional safeguard.

Reply:
We haven't set this up again after merging `dev/graph`, but it would happen on every merge into `main`. If we start doing something like per-PR staging environment we'd be creating/scanning images much more frequently